### PR TITLE
Footer Component

### PR DESCRIPTION
**Pull request by iansamz**
As a visitor, I would like the site to have a simple footer that shows me the icon and the copyright.

NOTE: the branch needs to be merged into a new branch called "redesign" otherwise it will break the live site!

- [x] Add a footer on everypage
- [x] Create a footer component
- [x] Show the following text in the component "2022 https://github.com/smartcontract Inc"
Expectation:
Screenshot 2022-07-27 at 11 59 50

Resolves: #271 

Result:
Screenshot 2022-07-27 at 12 01 07

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styles from './Footer.module.css';
+import clsx from 'clsx';
+
+function Footer() {
+  return (
+    <footer className={styles.container}>
+      <img src="/images/logo-footer.png" alt="blockchain logo" />
+      <p className={clsx('body-short-02', styles.copyright)}>{new Date().getFullYear()} @Blockchain Education</p>
```
Comment (hdJerry) [inline src/components/Footer/Footer.jsx:8]: `body-short-02` in an old typography. you would notice we added a comment to separate old typography.

**Review by hdJerry — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (vyktoremario) [reply src/components/Footer/Footer.jsx]: But why isn't it removed from the redesign?
Are we keeping it there for any reason?

**Review by vyktoremario — APPROVED** [review]
LGTM
But listen to @hdJerry

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (hdJerry) [reply src/components/Footer/Footer.jsx]: because we are making changes gradually. I could break other pages.
@vyktoremario

**Review by hdJerry — APPROVED** [review]
Thanks

**Pull request merged**